### ISSUE TITLE
citrix_adc_nitro_request: Support idempotent api calls

### ISSUE THREAD

**Issue by wuestkamp** [citrix/citrix-adc-ansible-modules#126]
Would it be (or is it already) possible to support idempotent **citrix_adc_nitro_request** api calls like described here: https://www.citrix.com/content/dam/citrix/en_us/citrix-developer/documents/Netscaler/how-do-i-add-or-update-a-ns-resource-seamlessly-using-a-single-nitro-api.pdf

```
• URL:	http://10.102.216.119/nitro/v1/config/lbvserver?idempotent=yes
• HTTP	Method: POST
• Request	Headers
o Content-Type:	application/json
o X-NITRO-ONERROR:	exit
• Payload:
{"lbvserver":{"name":"lbv1","servicetype":"http","lbmethod":"roundrobin"}}
```
Thanks!

**Comment by autoalan** [comment]
@wuestkamp I just submitted #137 for this very issue.

**Comment by giorgos-nikolopoulos** [comment]
Fixed with #147 

See [here](https://github.com/citrix/citrix-adc-ansible-modules/blob/master/samples/nitro_request/add_server_idempotent.yaml) for an example.
